CatProducer/plugins/RecoEventInfoProducer.cc

```diff
@@ -120,11 +120,14 @@ void RecoEventInfoProducer::produce(edm::Event& event, const edm::EventSetup& ev
       const unsigned int trigIndex = hltConfig_.triggerIndex(trigName);
       if ( trigIndex < hltHandle->size() )
       {
-        if ( hltHandle->accept(trigIndex) ) { isPassed = true; break; }
+	if ( hltHandle->accept(trigIndex) ) {
+	  const std::pair<int,int> prescales = hltConfig_.prescaleValues(event, eventSetup, trigName);
+	  psValue = prescales.first * prescales.second;
+	  break;
```

Comment:
i didnt add the break, it was there from before.

two different prescaled triggers never be saved in the one result.

first is the L1 and second is the HLT prescales.